var/spack/repos/builtin/packages/opa-psm2/package.py

```diff
@@ -12,6 +12,7 @@ class OpaPsm2(MakefilePackage):
     homepage = "http://github.com/01org/opa-psm2"
     url      = "https://github.com/01org/opa-psm2/archive/PSM2_10.3-8.tar.gz"
```

Comment:
It looks like these URLs now redirect to https://github.com/intel/opa-psm2. Can you update them?